.github/pull_request_template.md

```diff
@@ -4,20 +4,42 @@
 
 Fixes #
 
-# 👩🏻‍💻 What does this PR do? 
- <!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
+# 👩🏻‍💻 What does this PR do?
 
-# 🧪 How has/should this change been tested? 
-<!-- Explain how to setup for testing here if it is not already obvious, and how you've tested this PR. -->
+<!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
 
 ## 💌 Any notes for the reviewer?
-<!-- eg. Do you have any specific questions for the reviewer? Is there a high risk/complicated change they should focus on? If there are any general areas of the codebase your changes might have have touched or could cause side effects to, mention them here.-->
 
-## 📃 Documentation
+<!--
+eg. Do you have any specific questions for the reviewer? Is there a high risk/complicated change they should focus on? If there are any general areas of the codebase your changes might have have touched or could cause side effects to, mention them here.
+Anything half cooked but going to be finished off in a different PR?
+-->
+
+# 🧪 Testing
```

Comment:
Testing and set up sound the same 👀

Reply:
I agree it's pretty fuzzy... will draft merging them along with Mark's suggestions